packages/kubernetes-1.23/dockershim-symlink.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/bin/ln -sf /run/containerd/containerd.sock /run/dockershim.sock
```

Comment:
Nice, this is a pretty elegant way to do it. I like that it minimizes the delta between `kubelet.service` across variants.

Not a blocker, but you could potentially do the same thing with the `host-ctr` pull command. If you moved that to a drop-in then the hashes for `kubelet.service` would match across packages.

(I use `sha512sum packages/kubernetes-*/<file>` as a quick way to check for differences when reviewing these PRs. And yes, we should probably have a way to share these files between packages.)